vendors.json

```
@@ -148,5 +148,11 @@
     "constant": "TRAVIS",
     "env": "TRAVIS",
     "pr": { "env": "TRAVIS_PULL_REQUEST", "ne": "false" }
+  },
+  {
+    "name": "Netlify CI",
+    "constant": "NETLIFY",
+    "env": "NETLIFY_BUILD_BASE",
+    "pr": { "PULL_REQUEST": "true" }
```

The could be

```suggestion
    "pr": { "env": "PULL_REQUEST" }
```

Just to maintain consistency among env variables

---

@sibiraj-s We can not do that as PULL_REQUEST will be false for not pr .